src/test/java/io/github/mivek/parser/GeneralParseTest.java

```diff
@@ -35,7 +35,7 @@
             { "SCT026CB", true }, // Cloud
             { "ZZZ026CB", false }, // Cloud null
             { "+SHGSRA", true }, // Weather condition
-            { "+ZERT", false } // Weather null
+            { "+ZERT", true } // Weather null

```

Comment:
This one should remain false as `+` is not a valid weatherCondition and `ZERT` is not a valid phenomenon